wrs/index.js

```diff
@@ -1 +1,60 @@
 import './style.css';
+
+let leaderboard = [];
+
+const user = document.getElementById('Inputname');
+const score = document.getElementById('Inputscore');
+const pointsDeploy = document.querySelector('.table_container');
+
+const getScores = async () => {
+  try {
+    const response = await fetch('https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/TYBkzPuwqpwT7G0vx1Do/scores');
+    const json = await response.json();
+    leaderboard = json.result;
+    pointsDeploy.innerHTML = '';
+    leaderboard.forEach((dat) => {
+      const row = document.createElement('tr');
+      const name = document.createElement('td');
+      name.className = 'point';
+      name.textContent = `${dat.user}: ${dat.score}`;
+      row.appendChild(name);
+      pointsDeploy.appendChild(row);
+    });
+  } catch (error) {
+    // Handle the error
+  }
+};
+
+const addScore = async (userName, userScore) => {
+  try {
+    const response = await fetch('https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/TYBkzPuwqpwT7G0vx1Do/scores', {
+      method: 'POST',
+      body: JSON.stringify({
+        user: userName,
+        score: Number(userScore),
+      }),
+      headers: {
+        'Content-Type': 'application/json',
+      },
+    });
+    const json = await response.json();
+    if (json.response === 'Leaderboard score created.') {
+      getScores();
+    }
+    user.value = '';
+    score.value = '';
+  } catch (error) {
+    // Handle the error
+  }
```

Comment:
[OPTONAL] It will be a good UX if the score is automatically added to the score list without the need to click refresh. It would also be good if the score list is arranged in order of the scores (preferably) or in alphabetical order.